debezium-connector-sqlserver/src/test/java/io/debezium/connector/sqlserver/SqlServerChangeTableSetIT.java

```diff
@@ -203,6 +203,7 @@ private void addColumnToTable(boolean pauseAfterCaptureChange) throws Exception
         final int ID_START_4 = 10000;
         final Configuration config = TestHelper.defaultMultiDatabaseConfig()
                 .with(SqlServerConnectorConfig.SNAPSHOT_MODE, SnapshotMode.SCHEMA_ONLY)
+                .with(SqlServerConnectorConfig.MAX_TRANSACTIONS_PER_ITERATION, 1)
```

Comment:
Instead of changing existing tests, please introduce new ones.

Reply:
Adding this configuration to the test on the `master` branch indeed makes the test fail sometimes (4 of 5 in my case) with the "insufficient number of arguments" error. It seems to be good enough. It would be even better if a test failed consistently without the fix.